pontoon/translate/urls.py

```diff
@@ -0,0 +1,36 @@
+from django.conf import settings
+from django.conf.urls import url
+
+import views
+from . import URL_BASE
+
+
+urlpatterns = [
+    url(
+        r'^%s' % URL_BASE,
+        views.translate,
+        name='pontoon.translate',
+    ),
+]
+
+
+# For local development, we want to serve front-end static files from the
+# webserver that create-react-app is running. This way we benefit from
+# hot reloading and all the niceties of React dev tools.
+# To do that, we must overwrite the staticfiles serve view, and we also
+# want to catch websocket connexions.
+#
+# Note that because we override the static files serving, you will need to
+# run your local django server with the `--nostatic` option. That's
+# automatically done when running with `make run`.
+if settings.DEBUG:
```

Comment:
I'm a bit confused about that. On Socorro I only ever used `DEBUG`. What's the difference between the two?

Reply:
I see that we have `DEV` and `DEBUG` both set to `False` in prod and stage, and both set to `True` in local development settings. So it might be a good idea to join them.